src/resolvers/space-sx.ts

```diff
@@ -21,25 +20,22 @@ async function getSpaceProperty(key: string, url: string, property: 'avatar' | '
     ids.push(getAddress(key));
   }
 
-  const data = await axios({
-    url,
-    method: 'POST',
+  const {
     data: {
-      query: `
-        query {
-          spaces(where: { id_in: [${ids.map(item => `"${item}"`).join(', ')}] }) {
-            metadata {
-              ${property}
-            }
-          }
-        }`
-    },
-    ...axiosDefaultParams
-  });
-
-  const result = data.data?.data?.spaces
-    ?.map(space => space.metadata?.[property])
-    .filter(Boolean)[0];
+      data: { spaces }
+    }
```

Comment:
should be fine in case of an error?

Reply:
Mhhh it’s been tested only in case of no results

Reply:
For now, i think we should go back to the previous way `data.data?.data?.spaces` and we should be good to merge

Reply:
> Also, constructing the queries like this is a bit error prone:

What kind of errors you are referring to?